### PR TITLE
Write tests for product model

### PR DESCRIPTION
**Pull request by gamesfreak26**
Write a test for the product model.  Currently gets all products.

### CODE REVIEW

File: Models/product.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const mongoose = require("mongoose")
+const Schema = mongoose.Schema
+
+const Product = new Schema({
+    name: String,
+    description: String,
+    ownerId: { type: Schema.Types.ObjectId, ref: 'Owner' },
+    price: { type: Number, default: 0 },
+    category: [String],
```
Comment (rachelwong) [inline Models/product.js]: From my lack of knowledge around mongoose schemas, I'm not 100% sure whether `category` should be [String] or declared as an Array. 

`category: {
type: Array,
default: []
}`

or whether it should stay as it is. I am still referring to the mongoose documentation for guidance right now.

**Review by rachelwong — CHANGES_REQUESTED** [review]
I compared the schema against my final project at Coder. The schema looks good. 

My only uncertainty is around `category` as drop down field, as to whether this should be an `enum` or an `Array`.  This link says it should be an `enum`

https://stackoverflow.com/questions/61768692/how-to-set-mongoose-schema-for-select-option-list-and-set-property-on-selected-o

**Comment by gamesfreak26** [comment]
Made changes to the Model.  Changed it from a string to an Enum.

**Review by rachelwong — APPROVED** [review]
thank you Teej!

**Pull request merged**